api-specs/Gateway-EE/3.8/kong-ee.yaml

```
@@ -22,6 +22,7 @@ components:
       name: tags
       schema:
         type: string
+        minLength: 1
```

Ok this PR: https://github.com/Kong/docs.konghq.com/pull/8099 was a draft because I needed to revisit it as it was a bit inaccurate.

We need to change it to:
```suggestion
        nullable: false
```


According to these tests:

https://github.com/Kong/kong/pull/13723/files#diff-07c0e327c1c6351b4059a98d12a479f5807819c88d0a17cb119a1016ad6fe75eR76

A request like: `GET /consumers?tags=` fails with 400 (as it cannot be `null`) but a request like:
`GET /consumers?tags=''` should pass through as empty string is okay.

---

@nowNick can you rereview?